README.md

```diff
@@ -22,7 +22,8 @@ Supported domain zones:
 
 [API Referrence](https://unstoppabledomains.github.io/resolution/)
 
-# Installation
+## Installation
+
 Use the `npm` or `yarn` to install resolution.
```

Comment:
```suggestion
Use the `npm` or `yarn` to install the resolution package.

yarn global add @unstoppabledomains/resolution
npm install -g @unstoppabledomains/resolution
```

Reply:
This should trigger the binary installation process. Another option would be to install it inside the project which shouldn't install the binary at all.

Reply:
Good point, but I guess I'll mention this in the CLI section. I don't think many users want our library in their global environment